documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -649,21 +649,21 @@ where:
    :math:`[0,1,0]` respectively.
 
 - ``type``  Three character string denoting the the projection normalization of each
-  dimension, one character for each directions, e.g. ``'add'``, ``'arr'``, ``'adr'``.
+  dimension, one character for each directions, e.g. ``'add'``, ``'hrr'``, ``'adr'``.
 
-  At the moment there is only one possible option for the first (length) component of ``type``:
+  There are 6 possible options defining scale for the value of the momentum transfer:
 
-  1. ``'a'``     Inverse angstroms.
-
-  ..
-     2. ``'r'``
-
-        Reciprocal lattice units (r.l.u.) which normalises so that the maximum of
-        :math:`|h|`, :math:`|k|` and :math:`|l|` is unity.
-
-     3. ``'p'``
+  1. ``'a'``  :math:`|Q|` is measured in inverse angstroms.
```

Comment:
Just mod(Q)? I'd presume the whole vector is then in inv.Ang

Reply:
vector provides direction and its length measured in inverse angstrom. Seems logical